wurst/_handles/Item.wurst

```diff
@@ -65,6 +65,14 @@ public function item.isSellable() returns boolean
 public function item.isPawnable() returns boolean
 	return IsItemPawnable(this)
 
+/** Verifies whether item is not destroyed and has a valid type id. */
```

Comment:
I believe "verify" is the wrong word to use in both of these functions. 
They don't verify anything. Better change to something like:
`Returns true if the item is not destroyed and has a valid type id`
`Returns true if the item can be picked up by a unit at this moment`

Reply:
I still think `isPickupable` is a bit ambiguous to an item being pickupable by units in general, instead of being pickuable in its current state : )
But I guess it's fine.